src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const th = table.tHead.rows[0].cells[1];
+const targetTh = table.tHead.rows[0].cells[4];
```

Comment:
Similarly, using hardcoded indices like `cells[4]` can be problematic if the table structure changes. It might be better to dynamically determine the index based on some criteria.